README.md

```diff
@@ -103,3 +105,50 @@ spec:
 ```
 
 After a few minutes, operators will be upgraded automatically.
+
+### Test serverless-operator with Istio sidecar injection
+
+To install service mesh operator and enable istio sidecar injection, run `make install-mesh`
+
+```
+make install-mesh 
+```
+
+Then, create Knative Service with `sidecar.istio.io/inject: "true"` annotation:
+
+```sh
+cat <<EOF | oc apply -n default -f -
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello-example
+spec:
+  template:
+    metadata:
+      name: hello-example-1
+      annotations:
+        sidecar.istio.io/inject: "true"
+    spec:
+      containers:
+      - image: gcr.io/knative-samples/helloworld-go
+        name: user-container
+  traffic:
+  - latestRevision: true
+    percent: 100
+EOF
+```
+
+`ServiceMeshMemberRoll` created by `make install-mesh` includes following namespaces.
+
+- knative-serving
+- knative-serving-ingress
+- knative-eventing
+- default
```

Comment:
This is not accurate anymore I think?

Reply:
Sorry it remained previous setting. Updated.